packages/services/src/builder/index.ts

```diff
@@ -82,7 +82,11 @@ export class BuildManager {
         throw new ServerConnection.ResponseError(response, 'Build aborted');
       }
       if (response.status !== 200) {
-        const message = `Build failed with ${response.status}, please run 'jupyter lab build' on the server for full output`;
+        const message = `Build failed with ${response.status}.
+
+        If you are experiencing the build failure after installing an extension (or trying to include previously installed extension after updating JupyterLab) please check the extension website for new installation instructions as many extensions migrated to the prebuilt extensions system which no longer requires rebuilding JupyterLab (but uses a different installation procedure, typically involving a package manager such as 'pip' or 'conda').
```

Comment:
Agree that extensions can be distributed in many ways, but probably they will also be on public GitHub repositories?

```suggestion
        If you are experiencing the build failure after installing an extension (or trying to include previously installed extension after updating JupyterLab) please check the extension repository for new installation instructions as many extensions migrated to the prebuilt extensions system which no longer requires rebuilding JupyterLab (but uses a different installation procedure, typically involving a package manager such as 'pip' or 'conda').
```

Reply:
Well the only thing we know is they have an NPM website, which might be well outdated. I agree that repository is better because it should be up-to-date even if authors no longer publish to NPM.